### PR TITLE
docs: expand docs on meta `theme-color`

### PR DESCRIPTION
**Pull request by hasparus**
I've expanded the docs on meta theme color to present some more alternatives. 

---
 
 Further reading: https://css-tricks.com/meta-theme-color-and-trickery/

### CODE REVIEW

**Comment by lachlanjc** [comment]
>  Does anybody actually want a blue browser top bar?

Uhhh yeah I do :)

Maybe we just clarify you can use any of your colors depending on how bold a look you want?

**Comment by hasparus** [comment]
> Uhhh yeah I do :)

you monster xD

> Maybe we just clarify you can use any of your colors depending on how bold a look you want?

Definitely. I'll go back to it and try writing something more informative here.

**Review by lachlanjc — APPROVED** [review]

File: packages/docs/src/pages/guides/theme-color-meta-tag.mdx [hunk 1]
```diff
@@ -4,14 +4,23 @@ title: 'Theme Color Meta Tag'
 
 # Theme Color Meta Tag
 
-Use your framework’s method of adding a meta tag to `<head />`—[`react-helmet`](https://github.com/nfl/react-helmet) in Gatsby and [`next/head`](https://nextjs.org/docs/api-reference/next/head) in Next.js, for example.
+_Learn more about `<meta/>` theme color on
```
Comment (lachlanjc) [inline packages/docs/src/pages/guides/theme-color-meta-tag.mdx]:
```suggestion
_Learn more about `<meta />` theme color on
```

File: packages/docs/src/pages/guides/theme-color-meta-tag.mdx [hunk 2]
```diff
@@ -4,14 +4,23 @@ title: 'Theme Color Meta Tag'
...
+[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color)
+and [CSS Tricks](https://css-tricks.com/meta-theme-color-and-trickery/)._
+
+### with `useThemeUI`
+
+Grab your theme from the context with `useThemeUI` and use your framework's
+method of adding a meta tag to use your framework’s method of adding a meta tag
+to `<head />`
```
Comment (lachlanjc) [inline packages/docs/src/pages/guides/theme-color-meta-tag.mdx]: Duplicate text here! Might expand while we're at it:
```suggestion
Grab your theme from the context with `useThemeUI`, pick either a background
or accent color from the palette, then use your framework’s method of adding
a meta tag to add to `<head />`.
```

Comment (hasparus) [reply packages/docs/src/pages/guides/theme-color-meta-tag.mdx]: Thanks, mate! I love this new version.

**Comment by hasparus** [comment]
:rocket: PR was released in [`v0.14.2`](https://github.com/system-ui/theme-ui/releases/tag/v0.14.2) :rocket:

**Pull request merged**